src/aggregated_non_cooperative.jl

```diff
@@ -187,17 +188,15 @@ function finalize_results!(::AbstractGroupANC, ECModel::AbstractEC)
     final_step = field(gen_data, "final_step")
     n_steps = final_step - init_step + 1
     time_set = 1:n_steps
-
     project_lifetime = field(gen_data, "project_lifetime")
-    peak_categories = profile(market_data, "peak_categories")
 
-    # Set definitions
 
+    # Set definitions
+    user_set=ECModel.user_set
```

Comment:
we may add the space " = " so to keep the same format of the other.
As conventions, for inline comments, after the "#" I always add a space before the text, es "# SOME COMMENT".
Moreover, for inline comments after some code, I always add 2 spaces, es: "SOME CODE  # SOME COMMENT"